csharp/ql/src/experimental/Security Features/Serialization/XmlDeserializationWithDataSet.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name XML deserialization with a type type derived from DataSet or DataTable
+ * @description Making an XML deserialization call with a type derived from DataSet or DataTable types and may lead to a security problem. Please visit https://go.microsoft.com/fwlink/?linkid=2132227 for details."
+ * @kind problem
+ * @problem.severity error
+ * @precision medium
+ * @id cs/dataset-serialization/xml-deserialization-with-dataset
+ * @tags security
+ */
+
+import csharp
+import DataSetSerialization
+
+from UnsafeXmlReadMethodCall mc
+where exists(Method m | m.getACall() = mc)
```

Comment:
This line is not needed; each method call will always have a target.

Reply:
I understand now. I though that you only wanted me to get rid of it on the `from` clause. Fixing it.